Sources/Commands/PackageCommands/AddTarget.swift

```diff
@@ -60,6 +60,9 @@ extension SwiftPackageCommand {
         @Option(help: "The checksum for a remote binary target")
         var checksum: String?
 
+        @Option(help: "The testing library to use when generating test targets, which can be one of 'xctest', 'swift-testing', or 'none'")
+        var testingLibrary: PackageModelSyntax.AddTarget.TestHarness = .default
```

Comment:
Please break this up into two flags, `--enable/disable-xctest` and `--enable/disable-experimental-swift-testing`:

1. It would be consistent with the flags we've added to other commands;
2. swift-testing is experimental only;
3. The default behaviour if neither is specified should be to add support for XCTest only;
4. In the future, we can always change the default if needed; and
5. It is valid to specify _both_ libraries as dependencies for a test target.

Reply:
What you are proposing is a confusing design for functionality that is adding a brand-new test target. Roughly nobody is going to create a new test target that uses both of these testing libraries: they'll pick one and stick with it.

Other commands need to deal with one or both libraries being present, because existing test targets likely use XCTest and will want to pick up Swift Testing. This functionality is not for those use cases.